pkg/controller/authentication/configmap.go

```diff
@@ -45,40 +52,37 @@ func (r *ReconcileAuthentication) handleConfigMap(instance *operatorv1alpha1.Aut
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: consoleConfigMapName, Namespace: instance.Namespace}, consoleConfigMap)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			reqLogger.Error(err, "The configmap ", consoleConfigMapName ," is not created yet")
-			return err
-		} else {
-			reqLogger.Error(err, "Failed to get ConfigMap",  consoleConfigMapName)
+			reqLogger.Error(err, "The configmap ", consoleConfigMapName, " is not created yet")
 			return err
 		}
+		reqLogger.Error(err, "Failed to get ConfigMap", consoleConfigMapName)
+		return err
 	}
+
 	icpConsoleURL := consoleConfigMap.Data["MANAGEMENT_INGRESS_ROUTE_HOST"]
 
 	proxyConfigMapName := "ibmcloud-cluster-info"
 	proxyConfigMap := &corev1.ConfigMap{}
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: proxyConfigMapName, Namespace: instance.Namespace}, proxyConfigMap)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			reqLogger.Error(err, "The configmap ", proxyConfigMapName ," is not created yet")
-			return err
-		} else {
-			reqLogger.Error(err, "Failed to get ConfigMap",  proxyConfigMapName)
+			reqLogger.Error(err, "The configmap ", proxyConfigMapName, " is not created yet")
```

Comment:
@yannizhang2019 why did you remove the else block here?

Reply:
go lint flagged it, so I changed based on go lint recommendation, we still have a lot go lint errors